### ISSUE TITLE
Support templates monnaie, unité, nombre, nb

### ISSUE THREAD

**Issue by nico008** [spencermountain/wtf_wikipedia#286]
Hello,

Could you help adding support for these 2 templates?

https://fr.wikipedia.org/wiki/Mod%C3%A8le:Monnaie
This one looks very similar to https://en.wikipedia.org/wiki/Template:Currency which is already supported by wtf_wikipedia.
I tried to reuse a similar approach but I failed.

https://fr.wikipedia.org/wiki/Mod%C3%A8le:Unit%C3%A9
This one looks very complex but all I need for now is a usage similar to Currency.
In some cases, it has just one parameter. 
https://fr.wikipedia.org/wiki/Société_française_du_tunnel_routier_du_Fréjus
| chiffre d'affaires = {{Unité|110.80|M€}}
This template has 2 other synonyms: nombre and nb.
Example:
https://fr.wikipedia.org/wiki/Microsoft
| chiffre d'affaires = {{nombre|110.360 milliards d'euros}}

Thank you for your help!

**Comment by spencermountain** [comment]
added support for both in `7.4.1`

a year or so ago I ran through the whole english wikipedia dump and printed out [the most-common templates we're missing](https://github.com/spencermountain/wtf_wikipedia/wiki/Unsupported-Templates) - it may be a good idea to run that on fr and de wikipedias too. 

From my understanding, templates are perfectly independent between languages, and any resemblance between them is simply nice.

**Comment by nico008** [comment]
Hi Spencer,
I like this list for most-common templates missing!
Does this list include templates that are in infobox as well?
I was expecting to see {{nyse}} template and other stockexchange templates in the list.
Could you add instructions to generate this list so I could do the same for fr templates?
Thank you!

**Comment by spencermountain** [comment]
hey Nico, yeah, if I remember, I ran [dumpster-dive](https://github.com/spencermountain/dumpster-dive/) for a while, but I added a console.log statement to `./node_modules/wtf_wikipedia/src/templates/parse.js`. Then I just sorted them by frequency after.

It would be nice to see the results again, and also for `fr`. It's gotten a little more complicated, now that there's a fallback parser, which tries its best to parse the template, even though we don't explicitly know it - but still, the results would be wonderful to see
